mlflow/cli.py

```diff
@@ -510,8 +519,27 @@ def gc(backend_store_uri, run_ids):
         raise MlflowException(
             "This cli can only be used with a backend that allows hard-deleting runs"
         )
+
+    time_delta = 0
+
+    if older_than:
+        regex = re.compile(
+            r"^((?P<days>[\.\d]+?)d)?((?P<hours>[\.\d]+?)h)?((?P<minutes>[\.\d]+?)m)"
+            r"?((?P<seconds>[\.\d]+?)s)?$"
+        )
+        parts = regex.match(older_than)
+        if parts is None:
+            raise MlflowException(
+                "Could not parse any time information from '{}'. "
+                "Examples of valid strings: '8h', '2d8h5m20s', '2m4s'".format(older_than)
+            )
+        time_params = {name: float(param) for name, param in parts.groupdict().items() if param}
+        time_delta = int(timedelta(**time_params).total_seconds() * 1000)
+
+    deleted_run_ids_older_than = backend_store._get_deleted_runs(older_than=time_delta)
     if not run_ids:
-        run_ids = backend_store._get_deleted_runs()
+        run_ids = deleted_run_ids_older_than.copy()
```

Comment:
Is the copy necessary here?

Reply:
i removed the copy and just renamed the variable here